packages/tinacms/src/plugins/fields/numberFormat.ts

```diff
@@ -0,0 +1,19 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+export const parse = (value?: string) => value && +value;
```

Comment:
This file isn't being used

Reply:
It should be used. I import it in NumberFieldPlugin.tsx and use it there. Just like you do on the TextInput

Reply:
Hmm ... this seems strange. Seems like it has uploaded an older version. I try to push it again and see what happens.

Reply:
Should work now. Pushed it again. As you can see I'm using it in the NumberFieldPlugin.tsx now.